lib/interviewer/selectors/name-generator.js

```diff
@@ -23,13 +24,27 @@ const stageCardOptions = (_, props) => props.stage.cardOptions;
 const stageSortOptions = (_, props) => props.stage.sortOptions;
 const propPanels = (_, props) => props.stage.panels;
 
+const getIDs = createSelector(
+  propStageId,
+  getPromptId,
+  (stageId, promptId) => {
+    return {
+      stageId,
+      promptId,
+    };
+  },
+)
+
```

Comment:
A "helper" selector that just gets the stage and prompt ID and returns them in an object. This replaces the one I removed from the props selectors file, and puts it in a more appropriate location given that it now depends on state rather than props. I could also have put this in the interfaces selector file, and may do so if it turns out other interfaces use it.